### PR TITLE
Build issues

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: hardening.mk.inc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 CFORTIFY=-O2 -D_FORTIFY_SOURCE=2
-CASLR=-fPIC -fPIE
```
Comment (x1ddos) [inline hardening.mk.inc]: This wasn't even used and didn't realy make sense at all, mixing PIC with PIE.

**Comment by x1ddos** [comment]
Looking forward to a new release.

File: hardening.mk.inc (hunk 1 above)
Comment (benma) [reply hardening.mk.inc]: :+1:

**Comment by benma** [comment]
> I was able to run the appimage. Would've preferred the PIE and unrelated changes in separate PRs though.

The PR has two commits, one for each build issue. Not sure how two PRs would have helped?

Thanks for testing!

**Comment by x1ddos** [comment]
> The PR has two commits, one for each build issue. Not sure how two PRs would have helped?

Never mind. I think I'm too picky. It's just I find it useful to link to code reviews in other discussions. But this PR suddenly contains many other distracting changes and it makes it harder to follow because when I go to "Files changed" it shows a diff with all commits by default. We both know what is what but it might not be obvious to an occasional viewer. GitHub doesn't really matter though. Could've been an email thread with unrelated patches attached.

I shouldn't have mentioned libxcb troubles in that bug. :)

**Pull request merged**